### PR TITLE
Modify number() to support a range slider as an input type for a knob.

### PR DESCRIPTION
**Pull request by blorenz**
This PR adds the ability to represent a Knob for a number as a native browser input slider.

### CODE REVIEW

File: src/components/types/Number.js [hunk 1]
```diff
@@ -18,16 +18,30 @@ class NumberType extends React.Component {
...
-        id={knob.name}
-        ref="input"
-        style={styles}
-        value={knob.value}
-        type="number"
-        onChange={() => onChange(parseFloat(this.refs.input.value))}
-      />
-    );
+	  if (knob.range) {
+		  return (
```
Comment (arunoda) [inline src/components/types/Number.js]: create two methods returning these two types of input elements.
Then use a turnery to render these like:

````js
return knob.range ?  this.renderNormal() : this.randerRange();
````

File: src/index.js [hunk 2]
```diff
@@ -15,8 +15,17 @@ export function boolean(name, value) {
...
+  const defaults = {
+      range: false,
+      min: 0,
+      max: 10,
+      step: 1
+  };
+
+  const mergedOptions = {...defaults, ...options};
+
+  return manager.knob(name, { type: 'number', value, range: mergedOptions.range, min: mergedOptions.min, max: mergedOptions.max, step: mergedOptions.step});
```
Comment (arunoda) [inline src/index.js]: Create this options object separately and make it looks nice like this:

```js
const options = {
   ... mergedOptions,
   type: 'number',
   value,
}
```

File: README.md [hunk 3]
```diff
@@ -113,6 +113,21 @@ const defaultValue = 78;
...
+
+Allows you to get a number from the user using a range slider.
+
+```js
+const label = 'Temperature';
+const defaultValue = 73;
+const min = 60;
+const max = 90;
+const step = 1;
+const options = {range: true, min, max, step};
```
Comment (arunoda) [inline README.md]: Create the options object like this:

```js
const options = {
   range: true,
   min: 60,
   max: 90,
   step: 1,
};
```

**Comment by blorenz** [comment]
Thanks for the feedback @arunoda.  I applied those changes.  Since `options` was a redeclare of the parameter in src/index.js, I renamed your suggestion to `finalOptions`.

**Comment by arunoda** [comment]
Awesome. This is great.
We could ship this now.

**Comment by arunoda** [comment]
Released with `v1.7.0`.

**Pull request merged**